diff --git a/build/readConfigFileAndSetRootDir.js b/build/readConfigFileAndSetRootDir.js
index e989961ba71bbea43eb293ebdc7739f00f6e85e7..4e488db579a203b54413864e4d6fdc92819b5b40 100644
--- a/build/readConfigFileAndSetRootDir.js
+++ b/build/readConfigFileAndSetRootDir.js
@@ -153,7 +153,7 @@ async function readConfigFileAndSetRootDir(configPath) {
 // Load the TypeScript configuration
 const loadTSConfigFile = async configPath => {
   // Get registered TypeScript compiler instance
-  const registeredCompiler = await getRegisteredCompiler();
+  const registeredCompiler = await getRegisteredCompiler(configPath);
   registeredCompiler.enabled(true);
   let configObject = (0, _jestUtil().interopRequireDefault)(
     require(configPath)
@@ -167,15 +167,16 @@ const loadTSConfigFile = async configPath => {
   return configObject;
 };
 let registeredCompilerPromise;
-function getRegisteredCompiler() {
+function getRegisteredCompiler(configPath) {
   // Cache the promise to avoid multiple registrations
-  registeredCompilerPromise = registeredCompilerPromise ?? registerTsNode();
+  registeredCompilerPromise = registeredCompilerPromise ?? registerTsNode(configPath);
   return registeredCompilerPromise;
 }
-async function registerTsNode() {
+async function registerTsNode(configPath) {
   try {
     // Register TypeScript compiler instance
-    const tsNode = await import('ts-node');
+    const tsNodePath = require.resolve('ts-node', { paths: [configPath] })
+    const tsNode = await import(tsNodePath);
     return tsNode.register({
       compilerOptions: {
         module: 'CommonJS'
